docs/tutorials/deploy-model.md

```diff
@@ -0,0 +1,246 @@
+# Deploy a model
+
+```{admonition} Before you start...
+:class: note
+
+Please check our {ref}`"Before you start" guide<before-start>` to go over a few preliminary topics.
+```
+
+```{admonition} This tutorial was written for Jina 3.14
+:class: warning
+
+It will *probably* still work for later versions. If you have trouble, please ask on [our Slack](https://jina.ai/slack).
+```
+
+## Introduction
+
+In this tutorial we'll build a fast, reliable and scalable gRPC-based AI service. In Jina we call this an {class}`~jina.Executor`. Our Executor will use [Stable Diffusion](https://stability.ai/blog/stable-diffusion-public-release) to generate images from a given text prompt. We'll then use a {class}`~jina.Deployment` to serve it.
+
+![](images/deployment.png)
+
+```{admonition} Note
+:class: note
+
+A Deployment serves just one Executor. To use multiple Executors, read our {ref}`tutorial on building a pipeline<build-a-pipeline>`.
+```
+
+```{admonition} Run this tutorial in a notebook
+:class: tip
+
+You can also run this code interactively in [Colab](https://colab.research.google.com/github/jina-ai/jina/blob/master/.github/getting-started/notebook.ipynb#scrollTo=0l-lkmz4H-jW).
+```
+
+## Understand: Executors and Deployments
+
+- All data that goes in and out of Jina is in the form of a [DocumentArray](https://docarray.jina.ai/fundamentals/documentarray/) from the [DocArray](https://docarray.org) package.
+- An {ref}`Executor <executor-cookbook>` is a self-contained gRPC microservice that performs a task on a DocumentArray. This could be very simple (like merely capitalizing the entire text of a Document) or a lot more complex (like generating vector embeddings for a given piece of content).
+- A [Deployment]() lets you serve your Executor, scale it up with replicas, and allow users to send and receive requests.
```

Comment:
are we missing a link here ?

Reply:
Added link in [`e1b0970` (#5766)](https://github.com/jina-ai/jina/pull/5766/commits/e1b09701bc4e3fe376bbf668d1b7c2b27ed3adfe)